x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/findings_flyout.tsx

```diff
@@ -51,6 +52,39 @@ interface FindingFlyoutProps {
   findings: CspFinding;
 }
 
+const Cards = ({ data }: { data: Card[] }) => (
+  <EuiFlexGrid direction="column" gutterSize={'l'}>
+    {data.map((card) => (
+      <EuiFlexItem key={card.title} style={{ display: 'block' }}>
+        <EuiCard textAlign="left" title={card.title} hasBorder>
+          <EuiDescriptionList
+            compressed={false}
+            type="column"
+            listItems={card.listItems.map((v) => ({ title: v[0], description: v[1] }))}
+            style={{ flexFlow: 'column' }}
+            descriptionProps={{
+              style: { width: '100%' },
+            }}
+          />
+        </EuiCard>
+      </EuiFlexItem>
+    ))}
+  </EuiFlexGrid>
+);
+
+const FindingsTab = ({ tab, findings }: { findings: CspFinding; tab: FindingsTab }) => {
+  switch (tab) {
+    case 'remediation':
+      return <Cards data={getRemediationCards(findings)} />;
+    case 'resource':
+      return <ResourceTab data={findings} />;
+    case 'general':
+      return <Cards data={getGeneralCards(findings)} />;
+    default:
+      assertNever(tab);
+  }
+};
```

Comment:
the only change here is in line 80, the rest is completely the same, I just moved it